### DISCUSSION TITLE
Discussion number is generated even number.

### DISCUSSION THREAD

**Discussion by younho9** [community/community#11750] [Discussions]
Even though I started a first discussion, the discussion number is `#2`, and then next is `#4` and `#6`... 

Why are Discussion numbers generated only by even numbers?

**Comment by kecrily** [comment]
I guess you may not have noticed that issues, PR and discussion use the same set of sequences. If you've submitted an issue or PR in between discussions, it won't be continuous.

**Reply by younho9** [reply]
I created repository to reproduce https://github.com/younho9/reproduce-discussion-even-number/discussions/2

Here are no other issues, but discussion number is started from two. But, the next discussion continues normally...
